src/cockpit/superuser.py

```diff
@@ -220,11 +220,12 @@ def shutdown(self):
 
     # Connect-on-startup functionality
     def init(self, params: Dict[str, Union[bool, str, Sequence[str]]]) -> None:
-        name = params.get('id', 'any')
-        assert isinstance(name, str)
-        responder = AuthorizeResponder(self.router)
-        self._init_task = asyncio.create_task(self.go(name, responder))
-        self._init_task.add_done_callback(self._init_done)
+        if self.current != "root":
```

Comment:
That's a bit "meh" indeed.

Reply:
We could move this to bridge.py, and have that skip all SuperuserRoute related stuff for a privileged bridge.